pymetis/metis/CType.py

```diff
@@ -0,0 +1,2 @@
+RM = 0
```

Comment:
This doesn't make sense IMO. These values should come from Metis. If we include redundant definitions, we'll get confusing errors if Metis decides to change them. They make no promise that the numerical values will remain stable.

Reply:
I agree. I will move this part to the pybind11 wrapper.